### PR TITLE
Added Travis-ci and  pypi badges #6825

### PR DESCRIPTION
**Pull request by vishu1993**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,13 +3,11 @@ Nereid Image Transformation Module
...
 This module works as a dynamic image resizing addon for the statifc file
 functionality in Nereid.
-
-.. image:: https://travis-ci.org/openlabs/nereid-image-transformation.png?branch=develop
-  :target: https://travis-ci.org/openlabs/nereid-image-transformation
-
-.. image:: https://coveralls.io/repos/openlabs/nereid-image-transformation/badge.png
-  :target: https://coveralls.io/r/openlabs/nereid-image-transformation
-
+[![Build Status](https://travis-ci.org/openlabs/nereid-image-transformation.svg?branch=develop)](https://travis-ci.org/openlabs/nereid-image-transformation)
```
Comment (PritishC) [inline README.md]: One blank line.

File: README.md [hunk 2]
```diff
@@ -3,12 +3,11 @@ Nereid Image Transformation Module
 
 This module works as a dynamic image resizing addon for the statifc file
 functionality in Nereid.
-
-.. image:: https://travis-ci.org/openlabs/nereid-image-transformation.png?branch=develop
-  :target: https://travis-ci.org/openlabs/nereid-image-transformation
-
-.. image:: https://coveralls.io/repos/openlabs/nereid-image-transformation/badge.png
-  :target: https://coveralls.io/r/openlabs/nereid-image-transformation
+[![Build Status](https://travis-ci.org/openlabs/nereid-image-transformation.svg?branch=develop)](https://travis-ci.org/openlabs/nereid-image-transformation)
```
Comment (PritishC) [inline README.md]: Blank line here.

File: README.md [hunk 3]
```diff
@@ -3,12 +3,11 @@ Nereid Image Transformation Module
 
 This module works as a dynamic image resizing addon for the statifc file
 functionality in Nereid.
```
Comment (PritishC) [inline README.md]: Blank line after this line.

**Comment by PritishC** [comment]
Remove the `::` at lines 36 and 44.

**Comment by PritishC** [comment]
Remove any extra lines from the places where you removed the colons.

**Comment by PritishC** [comment]
LGTM.

**Comment by prakashpp** [comment]
@rituparnapanda please merge

**Pull request merged**